### ISSUE TITLE
[ISSUE for v2]:

### ISSUE THREAD

**Issue by Cheppi514** [w-okada/voice-changer#1565]
### Voice Changer Version

vcclient_win_std_2.0.78-beta

### Operational System

windows 11

### GPU

rx 6600

### CUDA Version

none

### Read carefully and check the options

- [ ] If you use win_cuda_torch_cuda edition, setup cuda? [see here](https://onnxruntime.ai/docs/execution-providers/CUDA-ExecutionProvider.html#requirements)
- [ ] If you use win_cuda edition, setup cuda and cudnn? [see here](https://onnxruntime.ai/docs/execution-providers/CUDA-ExecutionProvider.html#requirements)
- [ ] If you use mac edition, client is not launched automatically. Use chrome to open application.?
- [x] I've tried to change the Chunk Size
- [x] I've tried to set the Index to zero
- [x] I've read the [tutorial](https://github.com/w-okada/voice-changer/blob/master/tutorials/tutorial_rvc_en_latest.md)
- [x] I've tried to extract to another folder (or re-extract) the .zip file

### Does pre-installed model work?

No

### Model Type

RVC

### Issue Description

i have a problem with the ai voice i can hear my self but it either very low, faint or not processing at all. can you help me?

### Logs on console

C:\Users\Lucas\OneDrive\Desktop\voicechanger.ai\vcclient_win_std_2.0.78-beta (1)\dist>main.exe cui --https false --no_cui True
checking the modules...

                        vcclient

----------------------------------------------------------
 Application | http://localhost:18000
 Log(rich)   | http://localhost:18000/?app_mode=LogViewer
 Log(text)   | http://localhost:18000/vcclient.log
 API         | http://localhost:18000/docs
 License(js) | http://localhost:18000/licenses-js.json
 License(py) | http://localhost:18000/licenses-py.json
----------------------------------------------------------
Please press Ctrl+C once to exit vcclient.
Torch not compiled with CUDA enabled
torch\nn\utils\weight_norm.py:134: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.
  WeightNorm.apply(module, name, dim)
torch\nn\utils\weight_norm.py:25: UserWarning: The operator 'aten::_weight_norm_interface' is not currently supported on the DML backend and will fall back to run on the CPU. This may have performance implications. (Triggered internally at C:\__w\1\s\pytorch-directml-plugin\torch_directml\csrc\dml\dml_cpu_fallback.cpp:17.)
  return _weight_norm(v, g, self.dim)
Torch not compiled with CUDA enabled
Torch not compiled with CUDA enabled
